package.json

```diff
@@ -23,14 +23,14 @@
     "loader-utils": "^1.0.2",
     "object-assign": "^4.0.1",
     "object-hash": "^1.1.4",
-    "rimraf": "^2.6.1"
+    "rimraf": "^2.6.1",
+    "webpack": "^4.0.0"
```

Comment:
Should not be added as a dep, just devDep

Reply:
Oh, I forgot the flag when I install the webpack. I fixed it :)